Mathlib/Algebra/Star/CHSH.lean

```diff
@@ -164,14 +156,14 @@ we prepare some easy lemmas about √2.
 
 -- This calculation, which we need for Tsirelson's bound,
 -- defeated me. Thanks for the rescue from Shing Tak Lam!
-theorem tsirelson_inequality_aux : √2 * √2 ^ 3 = √2 * (2 * √2⁻¹ + 4 * (√2⁻¹ * 2⁻¹)) := by
+theorem tsirelson_inequality_aux : √(2 : ℝ) * √2 ^ 3 = √2 * (2 * (√2)⁻¹ + 4 * ((√2)⁻¹ * 2⁻¹)) := by
```

Comment:
It would be nice if `√2` elaborated to a `Real` in the absense of any other information, in the same way that `2` is by default a nat...

Reply:
And it would be nice to have a delaborator.

Reply:
Resolved by adding `default_instance` and an unexpander.